### PR TITLE
Hw1

### PR DESCRIPTION
**Pull request by ry2272**

### CODE REVIEW

**Comment by waxla** [comment]
Could convert the box-plot into a histogram to show the difference between genders, so that everything could be clearer.

Better rename your x axis labels.

Also, better adjust the sex legend, from a continuous one to a discrete one. 

Generally, good job!!

File: hw1/_posts/2014-09-18-hw1-Ruowei-Yang.Rmd [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+published: false
+tags: hw1
+---
+###Data Discription <br>
+The dataset concerns a study of teenage gambling in Britain. It has five variables, sex, status, income, verbal and gamble. Status is the socioeconomic status score based on parents' occupation, verbal is the verbal score in words out of 12 correctly defined which could indicate the teenager's education level. Gamble is the expenditure on gambling in pounds per year. 
+<br> <br>
+
+###Get Data
+```{r}
+library(faraway)
```
Comment (malecki) [inline hw1/_posts/2014-09-18-hw1-Ruowei-Yang.Rmd:15]: did you use this in another class? What does it add to base R that you need here? just the dataset?

File: hw1/_posts/2014-09-18-hw1-Ruowei-Yang.Rmd [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+attach(teengamb)
```
Comment (malecki) [inline hw1/_posts/2014-09-18-hw1-Ruowei-Yang.Rmd:16]: attach() is quite dangerous and should be avoided!

File: hw1/_posts/2014-09-18-hw1-Ruowei-Yang.Rmd [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+<p>
+
+From the boxplot, we can see that people with lower verbal score tends to spend more money on gambling which indicates tnat people with high education tend to spend less. Among the people with same verbal score, male tends to spend more than female.
+<br> <br>
+<p align="center">
+###Line charts of gamble expense for status and income by sex
+```{r, echo=FALSE}
+data1=teengamb[,c("status", "gamble", "sex")]
+data2=aggregate(gamble~status+sex, data=data1, FUN="mean")
+ggplot(data2, aes(x=status, y=gamble, group=sex))+geom_line(aes(colour = sex))
```
Comment (malecki) [inline hw1/_posts/2014-09-18-hw1-Ruowei-Yang.Rmd:41]: sex is being treated as numeric here and not a factor; although gender (and arguably sex) are a continuum, that’s probably not the intention in the dataset…

File: hw1/_posts/2014-09-18-hw1-Ruowei-Yang.Rmd [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+```
+<p>
+
+From these two plots, we can see that in both cases female tends to spend much less money on gambling than male. It is interesting to see that people with higher socialeconomic status spend less while people with higher income spend more.
+<br> <br>
+
+###Create a Linear Model
+```{r}
+gamblemod=lm(gamble~sex+status+income+verbal)
+summary(gamblemod)
```
Comment (malecki) [inline hw1/_posts/2014-09-18-hw1-Ruowei-Yang.Rmd:55]: One handy tool in rmarkdown is 'kable' which makes a sortable nice modern html table of stuff. I don't know if it has a method for lm output, but the dump of the summary thing is never very pretty.

**Pull request merged**